pymisp/api.py

```diff
@@ -368,6 +368,19 @@ def change_threat_level(self, event, threat_level_id):
 
         return self._check_response(response)
 
+    def _valid_uuid(self,uuid):
+        """ 
+            Test valid uuid générate by CakePhp
+
+            CakeText::uuid follow RFC 4122
+             - the third group must start with a 4,
+             - the fourth group must start with 8, 9, a or b.
+             
+             :param uuid: A UUID to validate
+        """
+        regex = re.compile('^[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}\Z', re.I)
+        match = regex.match(uuid)
+        return bool(match)
```

Comment:
Your right,

I refere to this stackoverflow for this job : 
http://stackoverflow.com/questions/11384589/what-is-the-correct-regex-for-matching-values-generated-by-uuid-uuid4-hex

Your solution seems more elegant  ^^